<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/img/icon/icon.png</normaloff>:/img/icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>256</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_ref_pushButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>560</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>add reference image</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="lineartwidget" native="true">
       <widget class="QLabel" name="sketch_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>256</width>
          <height>256</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../images/images.qrc">:/img/example/sketch.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="scriblewidget" native="true">
       <widget class="QLabel" name="scrible_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>256</width>
          <height>256</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../images/images.qrc">:/img/example/sketch.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="genwidget" native="true">
       <widget class="QLabel" name="gen_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>256</width>
          <height>256</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="gen_image_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>gen image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="add_scrible_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>add scrible image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="add_line_art_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>add line art image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionAdd_LineArt"/>
    <addaction name="actionAdd_Scribble"/>
    <addaction name="actionAdd_reference"/>
    <addaction name="actionAdd_ColorModel"/>
    <addaction name="actionSave_gen_image"/>
    <addaction name="actionSave_scribble"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_LineArt">
   <property name="text">
    <string>Add Line Art</string>
   </property>
  </action>
  <action name="actionAdd_Scribble">
   <property name="text">
    <string>Add Scribble</string>
   </property>
  </action>
  <action name="actionAdd_reference">
   <property name="text">
    <string>Add reference</string>
   </property>
  </action>
  <action name="actionSave_gen_image">
   <property name="text">
    <string>Save gen  image</string>
   </property>
  </action>
  <action name="actionSave_scribble">
   <property name="text">
    <string>Save scribble</string>
   </property>
  </action>
  <action name="actionAdd_ColorModel">
   <property name="text">
    <string>Add ColorModel</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
